src/components/new-conversation-dialog/create-group/index.js

```diff
@@ -189,12 +190,31 @@ const CreateGroup = ({
             description: 'description for group members input',
           })}
           maxHeight={67}
-          tags={selectedMembers}
           onFocus={onTagsInputFocus}
           onRemove={onMemberRemove}
           isNecessityLabel
           className={classes.members}
-        />
+        >
+          {selectedMembers.map(tag => {
```

Comment:
very small remark: I would call the iteration variable `member` and not `tag` because we are iterating over member objects here and will return Tags